library/firewall_lib.py

```diff
@@ -222,6 +224,48 @@
 except ImportError:
     HAS_FIREWALLD = False
 
+try:
+    from firewall.core.fw_nm import (
+        nm_is_imported,
+        nm_get_connection_of_interface,
+        nm_get_zone_of_connection,
+        nm_set_zone_of_connection,
+        nm_get_interfaces,
+        nm_get_client,
+    )
+
+    NM_IMPORTED = nm_is_imported()
```

Comment:
Is there a benefit to calling this function rather than just setting it to true?

Reply:
The function actually checks if Network Manager itself is imported in fw_nm.py, which is not checked by importing the functions. If the import fails, we can just assume no, but if it succeeds, the function call here I imagine would be better than calling `nm_is_imported()` for each zone changed.
https://github.com/firewalld/firewalld/blob/master/src/firewall/core/fw_nm.py#L31